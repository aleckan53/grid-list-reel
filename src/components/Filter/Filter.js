import React from 'react';
import styles from './css/Filter.module.css';

export default function Filter() {

  return (
    <div className={styles.filter}>
    </div>
  )
}