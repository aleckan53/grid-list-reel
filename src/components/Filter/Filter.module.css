/* No CSS *//*# sourceMappingURL=Filter.module.css.map */