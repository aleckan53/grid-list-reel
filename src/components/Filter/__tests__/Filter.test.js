import React from 'react';
import ReactDOM from 'react-dom';
import Filter from '../Filter'

describe('Filter component', () => {
  it('matches the snapshot', () => {
    const root = document.createElement('div');
    ReactDOM.render(<Filter/>, root);
    expect(root).toMatchSnapshot();
    ReactDOM.unmountComponentAtNode(root);
  });
});