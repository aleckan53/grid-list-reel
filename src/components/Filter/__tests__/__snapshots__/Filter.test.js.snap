// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter component matches the snapshot 1`] = `
<div>
  <div
    class="filter"
  >
    Filter
  </div>
</div>
`;
